src/models.ts

```diff
@@ -99,9 +101,10 @@ export const SortFieldDisplayName: {
 };
 
 export const DefaultSortDirection: {
-  [key in SortField]: SortDirection;
+  [key in SortField]: SortDirection | null;
 } = {
-  relevance: 'desc', // Can't actually change the sort direction for relevance
+  default: null,
+  relevance: null, // Can't actually change the sort direction for relevance
```

Comment:
q: if this changed to null, does that imply that it can change? the comment says it cannot change value?

Reply:
Oops, sounds like my comment wasn't clear enough! It's supposed to allude to how we disable the sort direction button when Relevance sort is selected -- i.e., the direction is not _user_-changeable in that case. So the `'desc'` value didn't actually do anything here, a potential source of confusion.

I can update to word this better. And FWIW, I have another in-flight PR that gathers up these various sort-related objects into a single declarative config which should hopefully be simpler to work with.